src/js/components/PageContent/index.d.ts

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { BoxProps } from '../Box/index';
+import { ColorType } from '../../utils';
+
+export interface PageContentProps {
+  background?:
+    | string
+    | {
+        color?: ColorType;
+        dark?: boolean | string;
+        image?: string;
+        position?: string;
+        opacity?: 'weak' | 'medium' | 'strong' | number | boolean;
+        repeat?: 'no-repeat' | 'repeat' | string;
+        size?: 'cover' | 'contain' | string;
+        light?: string;
+        fill?: 'horizontal';
+      };
+}
+
+export interface PageContentExtendedProps
+  extends PageContentProps,
+    Omit<BoxProps, 'background'> {}
```

Comment:
Actually, can we skip this `Omit` and just accept all BoxProps?

Reply:
I wanted to allow passing `background={{ fill: 'horizontal' }}` which is why I omitted the background prop from BoxProps since it doesn't include `fill`

Reply:
Is there a way to `Extend` rather than `Omit`? I'm just wondering how to avoid duplicating the BackgroundType and missing changes to it here.